nvi

```diff
@@ -10,7 +10,12 @@ N=$(tput sgr0) # Normal
 U=$(tput smul) # Underline
 
 print_usage() {
-  cat << EOF
+  less \
+    --raw-control-chars \
+    --no-init \
+    --quit-if-one-screen \
+    --QUIT-AT-EOF \
```

Comment:
Quitting at EOF seems strange to me. There's usually two ways my tools handle my requests for either their help text or their man page:

1.  Just print the full help text and exit. No `less`-like functionality at all.

    **Examples**:

    *   `docker --help`
    *   `git --help`
    *   `rm --help`
2.  Do like the regular man pages and work somewhat like `less` without the `--QUIT-AT-EOF` flag.

    **Examples**:

    *   `git help commit`
    *   `man docker`

Your suggestion here seems to be a mix of the two.